Source/DafnyCore/Backends/CSharp/CsharpCodeGenerator.cs

```diff
@@ -1781,7 +1781,7 @@ protected override string TypeName_UDT(string fullCompileName, List<TypeParamete
       return s;
     }
 
-    protected override string TypeName_Companion(Type type, ConcreteSyntaxTree wr, IToken tok, MemberDecl/*?*/ member) {
+    internal override string TypeName_Companion(Type type, ConcreteSyntaxTree wr, IToken tok, MemberDecl/*?*/ member) {
```

Comment:
Changing the visibility here because I needed to access this method to emit the proper path here.
https://github.com/dafny-lang/dafny/pull/5910/files#diff-88e28efa63a86a01c970f8732d7414bb1f8c5bcb77cbba8e4ae7e7b7bad18711R69
It's very Dafny-compiler specific but I could not change the visibility to anything less than internal to make it work.
I could have created another method that calls the protected one, but at this point I think it would have created redundancy for nothing.